### PR TITLE
[ticket/13180] Increase dateformat field size

### PR DESCRIPTION
**Pull request by hubaishan**
[ticket/13180] Increase dateformat field size

change field size to 64 in these files
add new migration file to update the database schema

PHPBB3-13180

### CODE REVIEW

File: phpBB/phpbb/db/migration/data/v31x/increase_size_of_dateformat.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	{
+		return array(
+			'change_columns' => array(
+				$this->table_prefix . 'users' => array(
+					'user_dateformat' => array('VCHAR:64', 'd M Y H:i'),
+				),
+			),
+		);
+	}
+
```
Comment (marc1706) [inline phpBB/phpbb/db/migration/data/v31x/increase_size_of_dateformat.php]: Please remove that empty line.

**Comment by hubaishan** [comment]
The commit failed  only in (php 5.3.3 travis test) but I don't know what is the problem?!!

**Comment by marc1706** [comment]
The failure on PHP 5.3.3 builds is currently caused by travis itself. Please ignore it.

**Comment by marc1706** [comment]
Closed in favor of #4112

**Pull request closed without merging**